<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{  background-color:#fff; }
#frame{  background-color:#03A9F4;
	box-shadow: 10px 5px 5px black;}
#label{ color:#fff; 	font-weight: bold; }



#Fondo{
	background-color: #fff;
}
#fondoTitulo{
	background-color:#03A9F4;
	box-shadow: 10px 5px 5px black;
}


#titulo{
	font-size: 25px;
    color:#fff;
	font-family: Arial, Verdana;
	font-weight: bold;
}

#btnBuscar{ background: #2E7D32; color: #fff; font-size: 12px; font-weight: bold; }
#btnGuardar{ background: #2E7D32; color: #fff; font-size: 12px; font-weight: bold; }
#btnEliminar{ background: #B71C1C; color: #fff; font-size: 12px; font-weight: bold; }
#btnLimpiar{ background: #0277BD; color: #fff; font-size: 12px;font-weight: bold;  }
#btnBuscarImg{ background: #0277BD; color: #fff; font-size: 12px; font-weight: bold; }


#calendario2 QToolButton {
  	height: 36px;
  	width: 120px;
  	color: white;
  	font-size: 12px;
  	icon-size: 35px, 35px;
  	background-color:#00838f
  }

 #calendario2 QMenu {
  	width: 150px;
  	left: 20px;
  	color: white;
  	font-size: 14px;
  	background-color: rgb(100, 100, 100);
  }

</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicación con menú</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>141</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_2, #menu1, #menu2, #menu3, #menu4, #menu5{
	background-color:#2c333e; color:#fff; font-size: 12px; font-family:   	Arial,Verdana; font-weight: bold; border:0px;
}
#menu1:hover, #menu2:hover, #menu3:hover, #menu4:hover, #menu5:hover{
	background-color:#0cc597; color:#fff;
}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="menu1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Pacientes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menu2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Doctores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menu3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Citas Médicas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menu4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Módulo 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menu5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Módulo 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Desarrollado por:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="framePrincipal">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>1091</width>
     <height>531</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="contenidoPaciente" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1091</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QWidget" name="formLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="frmUsuarios_6">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnBuscar">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtBusq"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>156</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="frmUsuarios_4">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbxBusq">
        <item>
         <property name="text">
          <string>Cédula</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Apellidos</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nombres</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ciudad</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblBusq_2">
        <property name="text">
         <string>Buscar Por:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="listaEmpleados">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>1061</width>
       <height>361</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <property name="columnCount">
      <number>23</number>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cédula</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombres</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apelidos</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha de Nacimiento</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Edad</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string># Aportaciones</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Dirección 1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Dirección 2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Teléfono 1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Teléfono 2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Días Laborales</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Género</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nivel Académico</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string># Cuenta Bancaria</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipo de Discapacidad</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre Recomendado</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Teléfono Recomendado</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Celular Recomendado</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ciudad</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Foto</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>130997977</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnNuevo">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Nuevo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEliminar">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
